core/sql/udrserv/UdrCfgParser.cpp

```diff
@@ -44,22 +44,15 @@ NABoolean UdrCfgParser::cfgFileIsOpen(NAString &errorText)
 
    NABoolean envFound = FALSE;
 
-   if(cfgFileName = getenv("MXUDRCFG"))
+   if(cfgFileName = getenv("TRAFUDRCFG"))
    {
       envFound = TRUE;
-      UDR_DEBUG1("UdrCfgParser(): MXUDRCFG cfgFileName is %s", cfgFileName);
+      UDR_DEBUG1("UdrCfgParser(): TRAFUDRCFG cfgFileName is %s", cfgFileName);
    }
    else
    {
-     NAString s("c:/tdm_sql/udr/mxudrcfg");
-     char installdir[1024];
-     Lng32 resultlength = 0;
-     Int32 res = ComRtGetInstallDir(installdir, 1024, &resultlength);
-     if (res == 0)
-     {
-       s = installdir;
-       s += "/mxudrcfg";
-     }
+     NAString s(getenv("MY_SQROOT"));
```

Comment:
We are using Facebook's string implementation, which is defined at https://github.com/facebook/folly/blob/master/folly/FBString.h

I think it will be an empty string. I could not make that determination from FB source code. Will try it on code with a debugger.